classes/02class/exercises/c02-network04/albertoeks/README.md

```diff
@@ -0,0 +1,196 @@
+# c02-network04
+
+## Commands Execution Output
+
+- Commands for creating the subnets:
+
+```yaml
+# find VPC from previous exercise
+$ aws ec2 describe-vpcs
+{
+    "Vpcs": [
+        {
+            "CidrBlock": "172.31.0.0/16",
+            "DhcpOptionsId": "dopt-5e83a939",
+            "State": "available",
+            "VpcId": "vpc-c6aba3a1",
+            "OwnerId": "059642851259",
+            "InstanceTenancy": "default",
+            "CidrBlockAssociationSet": [
+                {
+                    "AssociationId": "vpc-cidr-assoc-6566930e",
+                    "CidrBlock": "172.31.0.0/16",
+                    "CidrBlockState": {
+                        "State": "associated"
+                    }
+                }
+            ],
+            "IsDefault": true
+        },
+        {
+            "CidrBlock": "10.0.0.0/16",
+            "DhcpOptionsId": "dopt-5e83a939",
+            "State": "available",
+            "VpcId": "vpc-0cd0952e1f57efa55",
+            "OwnerId": "059642851259",
+            "InstanceTenancy": "default",
+            "CidrBlockAssociationSet": [
+                {
+                    "AssociationId": "vpc-cidr-assoc-082d9379773a4ec47",
+                    "CidrBlock": "10.0.0.0/16",
+                    "CidrBlockState": {
+                        "State": "associated"
+                    }
+                }
+            ],
+            "IsDefault": false
+        }
+    ]
+}
+
+# VPC from previous exercise = vpc-0cd0952e1f57efa55
+# create first public subnet
+aws ec2 create-subnet --vpc-id vpc-0cd0952e1f57efa55 --cidr-block 10.0.0.0/19 --region ap-southeast-2 --availability-zone ap-southeast-2a
```

Comment:
Another way to create the subnet and add the tag:

```bash
aws ec2 create-subnet \
    --vpc-id vpc-0cd0952e1f57efa55 \
    --cidr-block 10.0.0.0/19 \
    --region ap-southeast-2 \
    --availability-zone ap-southeast-2a \
    --tag-specifications 'ResourceType=subnet,Tags=[{Key=Name,Value=public-a}]'
```

> Just note that the `ResourceType` at this time is `subnet`

Reply:
Cool, that's better, thanks!